internal/command/cliconfig/cliconfig.go

```diff
@@ -476,17 +496,16 @@ func cliConfigFile() (string, tfdiags.Diagnostics) {
 		}
 	}
 
-	log.Printf("[DEBUG] Attempting to open CLI config file: %s", configFilePath)
-	f, err := os.Open(configFilePath)
+	log.Printf("[DEBUG] Attempting to stat CLI config file: %s", configFilePath)
+	_, err := cl.Stat(configFilePath)
```

Comment:
This changes the sematics slightly.  You need to check the stat result to ensure it's a file and not a directory.  I'm also not 100% sure, but I suspect that Open() also does a more comprehensive permission check?